packages/scalecube-microservice/src/api/Service.ts

```diff
@@ -1,6 +1,16 @@
 import { ServiceImplementation, ServiceDefinition } from '.';
 
+/**
+ * The information about service and the implementation for all the methods that are included in it
```

Comment:
```
/**
 * Service - definition and implementation of a service
 *
 * @param definition - Metadata of a service.
 * 
 *  Includes the name of a service and the map of methods that are included in it.
 *
 * @param reference - Implementation of a service.
 * 
 */
```

Reply:
definition and implementation of a service